playground/frontend/lib/pages/embedded_playground/widgets/embedded_actions.dart

```diff
@@ -40,19 +41,41 @@ class EmbeddedActions extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.all(kMdSpacing),
-      child: SizedBox(
-        width: kTryPlaygroundButtonWidth,
-        height: kTryPlaygroundButtonHeight,
-        child: Consumer<PlaygroundController>(
-          builder: (context, controller, child) => ElevatedButton.icon(
-            icon: SvgPicture.asset(kLinkIconAsset),
-            label: Text(AppLocalizations.of(context)!.tryInPlayground),
-            onPressed: () => _openStandalonePlayground(controller),
-          ),
+    return Row(
+      children: [
+        Consumer<PlaygroundController>(
+          builder: (_, controller, __) {
+            final selectedExample = controller.selectedExample;
+            final hasGithubLink = selectedExample?.urlVcs != null;
+            final hasColabLink = selectedExample?.urlNotebook != null;
+
+            return Row(
+              children: [
+                if (hasGithubLink)
+                  LinkButton.github(selectedExample?.urlVcs ?? ''),
+                if (hasColabLink) ...[
+                  const SizedBox(width: kMdSpacing),
```

Comment:
This spacing should not belong to any of the buttons.

Option 1:

https://github.com/alexeyinkin/mefolio-standalone/blob/ab95a0f8cca06c589a24f1ffe82b1c8bef847c86/flutter/lib/util/util.dart#L2

https://github.com/alexeyinkin/mefolio-standalone/blob/ab95a0f8cca06c589a24f1ffe82b1c8bef847c86/flutter/lib/pages/contact/widgets/contacts.dart#L23

Option 2: Wrap all buttons with padding.

Reply:
Paddings added